scrc/dataset_creation/judgment_dataset_creator.py

```diff
@@ -7,17 +7,22 @@
 from scrc.utils.main_utils import get_config
 from scrc.utils.sql_select_utils import convert_to_binary_judgments
 
+import os
+from tqdm import tqdm
+from root import ROOT_DIR
+from scrc.utils.court_names import get_all_courts, get_issue_courts, get_error_courts
+
 
 class JudgmentDatasetCreator(DatasetCreator):
     """
     Creates a dataset with the facts or considerations as input and the judgments as labels
     """
 
-    def __init__(self, config: dict):
+    def __init__(self, config: dict, debug: bool = True):
         super().__init__(config)
         self.logger = get_logger(__name__)
 
-        self.debug = True
+        self.debug = debug
```

Comment:
good idea. Please move this to the dataset creator.

Reply:
What do you mean exactly with 'move'?
As far as I know the `def __init__()` in the JudgmentDatasetCreator replaces the `__init__` from the parent class. That's why we use the `super().__init()` right. That's why I handed over the "debug"-parameter to the parent and implemented it in their __init__-method which looks like this:
`    def __init__(self, config: dict, debug: bool = True):`
`             super().__init__(config, debug)`

Is this what you meant?

Reply:
yes, exactly. Please remove it from inside all the dataset creators init functions, so that it is set only when calling the init function